anyway/infographics_utils.py

```diff
@@ -1785,6 +1801,8 @@ def localize_after_cache(request_params: RequestParams, items_list: List[Dict])
             )
         else:
             logging.error(f"localize_after_cache: bad input (missing 'name' key):{items}")
+        if request_params.lang != "en":
+            items["meta"]["information"] = _(items["meta"]["information"])
```

Comment:
key error exception should occur since `items["meta"]["information"]` is initialized in class Widget. I noticed that if there is no translation - there is no error from flask_babel - so no need for try except, and I can even remove the if condition `if request_params.lang != "en"`
@ziv17 @BarVolunteering what do you think?

Reply:
Looks good!